docs/List.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: "The List View"
+title: 'The List View'
```

Comment:
Too many things have changed in this file. Can you reduce the changes to what you added?

Reply:
Sorry, vscode was running prettier on docs files